sdk/storage/azure-storage-blob-cryptography/src/test/java/com/azure/storage/blob/specialized/cryptography/EncryptedFlux.java

```diff
@@ -103,7 +105,7 @@ ByteBuffer limit() is exclusive, which is why we add one to a limit if that byte
 
         EncryptedBlob encryptedBlob = new EncryptedBlobAsyncClient(null, "https://random.blob.core.windows.net",
             BlobServiceVersion.getLatest(), null, null, blobName, null, null, null, key, "keyWrapAlgorithm", null,
-            EncryptionVersion.V1, false)
+            EncryptionVersion.V1, false, new BlobEncryptionOptions().setAuthenticatedRegionDataLength(GCM_ENCRYPTION_REGION_LENGTH))
```

Comment:
Once we add the default to the encryption options class, we should be able to remove setting the region length here.